From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Erik Corry <erikcorry@chromium.org>
Date: Mon, 3 Mar 2025 22:30:37 +0100
Subject: Modify where to look for fast_float and simdutf.

Similar to fp16, these dependencies now needs to be downloaded by bazel.

diff --git a/BUILD.bazel b/BUILD.bazel
index f19f85b5b94a36b0b00c37b075e6cc7291ee0b34..19d7d942c9dc49b48425285d92d6c93e27371ab9 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -2542,8 +2542,6 @@ filegroup(
         "src/zone/zone-type-traits.h",
         "src/zone/zone-utils.h",
         "third_party/rapidhash-v8/rapidhash.h",
-        "third_party/simdutf/simdutf.cpp",
-        "third_party/simdutf/simdutf.h",
         "third_party/siphash/halfsiphash.cc",
         "third_party/siphash/halfsiphash.h",
         "third_party/utf8-decoder/utf8-decoder.h",
@@ -4325,12 +4323,6 @@ cc_library(
     ],
 )
 
-cc_library(
-  name = "simdutf",
-  srcs = ["third_party/simdutf/simdutf.cpp"],
-  hdrs = ["third_party/simdutf/simdutf.h"],
-)
-
 v8_library(
     name = "v8_libshared",
     srcs = [
@@ -4359,9 +4351,9 @@ v8_library(
         ":noicu/generated_torque_definitions",
     ],
     deps = [
-        "//third_party/fast_float/src:fast_float",
+        "@fast_float",
         ":lib_fp16",
-        ":simdutf",
+        "@simdutf",
         ":v8_libbase",
         "@abseil-cpp//absl/container:btree",
         "@abseil-cpp//absl/container:flat_hash_map",
diff --git a/src/builtins/builtins-typed-array.cc b/src/builtins/builtins-typed-array.cc
index 19d9d3ba904bbaf791578035f65b225042fca590..f3c54b5c1c91545b97e2a38aa2933e495948ebf7 100644
--- a/src/builtins/builtins-typed-array.cc
+++ b/src/builtins/builtins-typed-array.cc
@@ -11,7 +11,7 @@
 #include "src/objects/heap-number-inl.h"
 #include "src/objects/js-array-buffer-inl.h"
 #include "src/objects/objects-inl.h"
-#include "third_party/simdutf/simdutf.h"
+#include "simdutf.h"
 
 namespace v8 {
 namespace internal {
